controllers/components/kserve/kserve_support.go

```diff
@@ -114,7 +114,6 @@ func defineServiceMeshFeatures(ctx context.Context, cli client.Client, dscispec
 					manifest.Location(Resources.Location).
 						Include(
 							path.Join(Resources.ServiceMeshDir, "activator-envoyfilter.tmpl.yaml"),
-							path.Join(Resources.ServiceMeshDir, "envoy-oauth-temp-fix.tmpl.yaml"),
```

Comment:
Do we need to cleanup existing RHOAI installation?

Reply:
i think it is fine to leave what it has been done.
it does not harm, but not needed for any newer version ossm (>=2.4) to have this patch.

we added it when 2.3 was still supported version, now even 2.4 will face EOL after next week.